pkg/kv/cosmosdb/store.go

```diff
@@ -129,6 +129,13 @@ func getOrCreateDatabase(ctx context.Context, client *azcosmos.Client, params *k
 }
 
 func getOrCreateContainer(ctx context.Context, dbClient *azcosmos.DatabaseClient, params *kvparams.CosmosDB) (*azcosmos.ContainerClient, error) {
+	var throughputProperties azcosmos.ThroughputProperties
+	if params.Autoscale {
+		throughputProperties = azcosmos.NewAutoscaleThroughputProperties(int32(params.Throughput))
+	} else {
+		throughputProperties = azcosmos.NewManualThroughputProperties(int32(params.Throughput))
+	}
```

Comment:
do we have to configure throughput? can we keep the current way as the default and not pass any? or passing nil is equal to NewManual...

Reply:
Done